Signal/translations/en.lproj/Localizable.strings

```diff
@@ -529,6 +529,15 @@
 /* message footer while attachment is uploading */
 "MESSAGE_STATUS_UPLOADING" = "Uploading...";
 
+/* Indicates that this 1:1 conversation has been blocked. */
+"MESSAGES_VIEW_CONTACT_BLOCKED" = "This User is Blocked";
+
+/* Indicates that a single member of this group has been blocked. */
+"MESSAGES_VIEW_GROUP_1_MEMBER_BLOCKED_FORMAT" = "1 Blocked Member of this Group";
```

Comment:
Maybe: Similarly, maybe "Blocked 1 Member of this Group" and "Blocked %d Members of this Group".

take it or leave it.